src/fides/core/config/database_settings.py

```diff
@@ -16,36 +16,77 @@
+    )
+    db: str = Field(
+        default="default_db", description="The name of the application database."
+    )
+    password: str = Field(
+        default="defaultpassword",
+        description="The password with which to login to the application database.",
+    )
+    port: str = Field(
+        default="5432",
+        description="The port at which the application database will be accessible.",
+    )
+    server: str = Field(
+        default="default-db",
+        description="The hostname of the application database server.",
+    )
+    task_engine_pool_size: int = Field(
+        default=50,
+        description="Number of concurrent database connections Fides will use for executing privacy request tasks, either locally or on each worker. Note that the pool begins with no connections, but as they are requested the connections are maintained and reused up to this limit.",
+    )
+    task_engine_max_overflow: int = Field(
+        default=50,
+        description="Number of additional 'overflow' concurrent database connections Fides will use for executing privacy request tasks, either locally or on each worker, if the pool reaches the limit. These overflow connections are discarded afterwards and not maintained.",
+    )
+    test_db: str = Field(
+        default="default_test_db",
+        description="Used instead of the 'db' value when the FIDES_TEST_MODE environment variable is set to True. Avoids overwriting production data.",
+        exclude=True,
+    )
+    user: str = Field(
+        default="defaultuser",
+        description="The database user with which to login to the application database.",
+    )
+
+    # These must be at the end because they require other values to construct
+    sqlalchemy_database_uri: str = Field(
+        default="",
+        description="Programmatically created connection string for the application database.",
+        exclude=True,
+    )
+    sqlalchemy_test_database_uri: str = Field(
+        default="",
+        description="Programmatically created connection string for the test database.",
+        exclude=True,
+    )
+    async_database_uri: str = Field(
+        default="",
+        description="Programmatically created asynchronous connection string for the configured database (either application or test).",
+        exclude=True,
+    )
+    sync_database_uri: str = Field(
+        default="",
+        description="Programmatically created synchronous connection string for the configured database (either application or test).",
+        exclude=True,
+    )
 
     @validator("sync_database_uri", pre=True)
     @classmethod
     def assemble_sync_database_uri(
         cls, value: Optional[str], values: Dict[str, str]
     ) -> str:
         """Join DB connection credentials into a connection string"""
-        if isinstance(value, str) and value != "":
-            # This validates that the string is a valid PostgresDns.
-            return str(PostgresDsn(value))
```

Comment:
Was the `PostgresDsn` validation causing `Field` errors or was it deemed no longer necessary?

Reply:
not working as we expected (didn't actually validate anything) 🤷 the error will get caught at runtime